package.json

```diff
@@ -80,7 +80,7 @@
     "zone.js": "^0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.803.19",
```

Comment:
Does this version need to be changed?

Reply:
this is not my change when i did rebase i got merge conflict and i resolved so its added to my change, this happened  because i was working on previous version.